selfdrive/common/modeldata.h

```diff
@@ -26,12 +26,24 @@ const mat3 fcam_intrinsic_matrix = (mat3){{
   0.0, 2648.0, 1208.0/2,
   0.0,   0.0,   1.0
 }};
+
+const mat3 ecam_intrinsic_matrix = (mat3){{
+  567.0, 0.0, 1928.0/2,
```

Comment:
the focal length is probably not exact since we are cropping without distortion remapping

Reply:
best bet is to just play with this until it matches the fcam fov seeing from the ui